types/carbon-components-react/lib/components/UIShell/Content.d.ts

```diff
@@ -1,7 +1,7 @@
 import * as React from "react";
-import { ReactCreateElementParam, FCReturn, FCProps } from "../../../typings/shared";
+import { ReactCreateElementParam, FCReturn, FCProps, ReactAttr } from "../../../typings/shared";
 
-export interface ContentPropsBase {
+export interface ContentPropsBase extends ReactAttr {
```

Comment:
One of the reasons I didn't include this on `ContentPropsBase` interface is that technically [React.createElement](https://reactjs.org/docs/react-api.html#createelement) can accept a tag name, a component, or fragment. However, it's probably overkill as carbon expects a tag name only and likely most if not everyone is not customizing the `tagName` from the default of `main`. 

I still think it would fit better as the default value of the generic parameter `E` on lines 10 and 12 below. I'm not sure why it wasn't this to begin with.. probably a mistake.

Reply:
@salvoravida